src/router/credentialRouter.ts

```diff
@@ -1,13 +0,0 @@
-import { Router } from "express";
-import { createCredential, getCredential, deleteCredential } from "../controllers/credentialController.js";
-import schemaValidator from "../middlewares/schemaValidator.js";
-import { credentialSchema } from "../schemas/credentialSchema.js";
-import { tokenValidator } from "../middlewares/tokenValidator.js";
-
-const credentialRouter = Router();
-
-credentialRouter.post("/create-credential", tokenValidator, schemaValidator(credentialSchema), createCredential);
```

Comment:
Não há necessidade de especificar na rota se é para criar colocando "create-credential", visto que se for POST...irá criar, GET... irá retornar, PUT... irá atualizar, DELETE... irá deletar.

Vale muito mais a pena você escolher um padrão de sempre  deixar o nome da entidade no plural, e só colocar parâmetros quando for necessário (como o id) ou quando precisar mudar de entidade ("users/:id/addresses")